packages/shaderlab/src/shaders/shadingPBR/LightIndirectPBR.glsl

```diff
@@ -74,7 +78,14 @@ void evaluateSpecularIBL(Varyings varyings, SurfaceData surfaceData, BRDFData br
         vec3 speculaColor = brdfData.specularColor;
     #endif
 
-    outSpecularColor += surfaceData.specularAO * radianceAttenuation * radiance * envBRDFApprox(speculaColor, brdfData.roughness, surfaceData.dotNV);
+    #ifdef MATERIAL_ENABLE_SHEEN
+    outSpecularColor *= brdfData.sheenScaling;
+    vec3 environmentBRDF = brdfData.sheenDFG * surfaceData.sheenColor;
```

Comment:
```
sheen_material = sheenColor * sheen_brdf + material * sheen_albedo_scaling
```
It is an additive relationship, not a choice between two options.

Reply:
outSpecularColor += surfaceData.specularAO * radianceAttenuation * radiance * environmentBRDF;
It will additive at the end of the output

Reply:
> ```
> sheen_material = sheenColor * sheen_brdf + material * sheen_albedo_scaling
> ```
> 
> It is an additive relationship, not a choice between two options.

    outSpecularColor += surfaceData.specularAO * radianceAttenuation * radiance * environmentBRDF;
It will additive at the end of the output